source/common/stats/symbol_table_impl.h

```diff
@@ -0,0 +1,99 @@
+#pragma once
+
+#include <algorithm>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "envoy/common/exception.h"
+#include "envoy/stats/symbol_table.h"
+
+#include "common/common/assert.h"
+
+#include "absl/strings/str_join.h"
+#include "absl/strings/str_split.h"
+
+namespace Envoy {
+namespace Stats {
+
+class SymbolTableImpl : public SymbolTable {
+public:
+  std::vector<Symbol> encode(const std::string& name) override {
```

Comment:
Why are all of these methods implemented in the header?

Reply:
No good reason -- I will move them to the `.cc` when responding to other incumbent reviews.

Reply:
SG, thanks.